--- language/en/common.php	2009-09-15 14:25:12.000000000 -0700
+++ language/en/common.php	2009-09-15 14:25:17.000000000 -0700
@@ -40,6 +40,8 @@
 //
 
 $lang = array_merge($lang, array(
+	'AMO_NO_NICKNAME' => 'Please <a href="https://addons.mozilla.org/users/edit">add a nickname to your addons.mozilla.org account</a> before using these forums.',
+
 	'TRANSLATION_INFO'	=> '',
 	'DIRECTION'			=> 'ltr',
 	'DATE_FORMAT'		=> '|d M Y|',	// 01 Jan 2007 (with Relative days enabled)
